js/core/src/registry.ts

```diff
@@ -291,9 +291,8 @@ export class Registry {
     const key = `/${type}/${action.__action.name}`;
     logger.debug(`registering ${key}`);
     if (this.actionsById.hasOwnProperty(key)) {
-      // TODO: Make this an error!
-      logger.warn(
-        `WARNING: ${key} already has an entry in the registry. Overwriting.`
+      logger.error(
+        `Failed to register ${key} action because it's already in the registry.`
```

Comment:
this message is not accurate -- `logger.error` does not throw an error, it just logs at "error" log level, so it would not fail to register a duplicate key. 

I'm not sure if we can actually make this throw an error at this point... I know for sure there would need to be a number of tests that will need fixing.

s/logger.warn/logger.error/ is a good idea and a low risk change.

Reply:
I see, I changed the message to `ERROR`.